test/verify/machineslib.py

```diff
@@ -1732,7 +1732,7 @@ def installWithErrorTest(dialog):
         self.machine.execute('rm $(which virt-install)')
         self.browser.reload()
         self.browser.enter_page('/machines')
-        self.browser.wait_visible("#create-new-vm:disabled")
```

Comment:
I would keep this as well. we are interested to check that the button is disabled aside from the testdata attribute.